.env

```diff
@@ -1,11 +1,11 @@
 PINPOINT_VERSION=2.0.4
 SPRING_PROFILES=release
 #zookeeper information required
-PINPOINT_ZOOKEEPER_ADDRESS=pinpoint-hbase
+PINPOINT_ZOOKEEPER_ADDRESS=zoo1
 
 ### Pinpoint-Hbase
 
-PINPOINT_HBASE_NAME=pinpoint-hbase
+PINPOINT_HBASE_NAME=zoo1
```

Comment:
I think this should stay as is. since the name of hbase container shouldn't chnage

Reply:
Oh, that was the reason why pinpoint-hbase show its container name as zoo1. I'll fix it.